www/nginx/src/opnsense/mvc/app/views/OPNsense/Nginx/index.volt

```diff
@@ -695,9 +838,31 @@
             </tfoot>
         </table>
     </div>
+    <div id="subtab_nginx-other-config-preview" class="tab-pane fade">
+        <div id="nginx_conf_container" class="nginx_table_responsive">
+            <pre style="margin: 0 0 0;"><table class="ngx_conf_table" id="nginx_conf">
+                <tbody class="ngx_conf_table_body"></tbody>
+            </table></pre>
+            <table class="table table-striped table-condensed">
+                <tbody>
+                    <tr>
+                        <td>
+                            <div id="config_help_text" style="display:none">
+                                {{ lang._("Configuration files may contain sensitive information, keep it safe.") }}
+                                <a id="nginx_config_copy">{{ lang._('Click here to copy to clipboard.') }}</a>
+                            </div>
+                            <div>
+                                <button class="btn btn-primary" id="conf_show_btn" data-type="config" type="button"><b>{{ lang._('Show Config') }}</b></button>
+                                <button class="btn btn-primary" id="conf_test_btn" data-type="test" type="button"><b>{{ lang._('Test Config') }}</b></button>
+                            </div>
```

Comment:
maybe you could move those buttons to somewhere else. For the plugin, the best section would be the top right service buttons. They are currently handled by core but maybe it would make sense to override it?

Maybe ask Franco about it.

Reply:
@fabianfrz @fichtner I just try to follow the style used in other places (in core or, for example, in the HAProxy plugin). I think users will search this buttons in this place )